ui-tests/tests/jupyter_scheduler.spec.ts

```diff
@@ -6,6 +6,24 @@ import { expect, test } from '@jupyterlab/galata';
  */
 test.use({ autoGoto: false });
 
-test('should emit an activation console message', async ({ page }) => {
-  // TODO: Add real tests
+test('Should add a Notebook Jobs card to JupyterLab launcher', async ({
+  page
+}) => {
+  await page.goto();
+  const launcher = page.locator('div[role="main"] >> text=Launcher');
+  await launcher.waitFor();
+  const jobsElement = page.locator('div.jp-LauncherCard[title="Notebook Jobs"]');
+
+  await expect(jobsElement).toBeVisible();
+});
+
+test('Test should fail', async ({
+  page
+}) => {
+  await page.goto();
+  const launcher = page.locator('div[role="main"] >> text=Launcher');
+  await launcher.waitFor();
+  const jobsElement = page.locator('div.jp-LauncherCard[title="Nonexisting title 2869291980"]');
```

Comment:
This seems like a system generated title, which might change in different runs. Is there another selector that will work consistently? Try this:

```ts
page.locator('div[role="main"] >> text="Notebook Jobs"')
```

Reply:
Oh this is a nonsensical title, I created this test to fail on purpose to test CI. This is not going into PR when it's ready.